server/templates/staff/course/students.list.html

```diff
@@ -21,6 +21,7 @@ <h3 class="box-title"><span> {{ title }} </span></h3>
                 <th class="sort" data-sort="role">Role</th>
                 <th class="sort" data-sort="timestamp">Enrolled At</th>
                 <th class="sort" data-sort="section">Section</th>
+                <th class="sort" data-sort="unenroll">Action</th>
```

Comment:
This column shouldn't be sortable. It does some weird stuff if you try to sort by this column